packages/core/src/lib/disk.ts

```diff
@@ -49,20 +57,23 @@ async function getReadableStreamFromFile(file: File) {
 export class DiskWrapper {
   private disk: Disk
 
-  constructor() {
-    this.disk = new Disk(this.buildDisk())
+  constructor(visibility: 'private' | 'public' = 'private') {
+    this.disk = new Disk(this.buildDisk(visibility))
```

Comment:
tbh this class would probably benefit from a refactor, it should receive an instance of Disk as a parameter an otherwise default to the local disk instance. Maybe leave a TODO?

Reply:
okay